src/api.rs

```diff
@@ -1101,6 +1108,13 @@ impl<T: Pixel> ContextInner<T> {
   }
 
   fn compute_lookahead_data(&mut self) {
+    /// Enable to make this code output:
+    /// - i-qres.png: quarter-resolution luma planes,
+    /// - i-hres.png: half-resolution luma planes,
+    /// - i-mvs.bin: motion vectors,
+    /// - i-imps.bin: block importances.
+    const WRITE_DEBUG_FILES: bool = false;
```

Comment:
make it a `#[cfg(feature=dump_image)]` or such.

Reply:
Leave the `const` or have enabling the feature dump the files unconditionally? I think I prefer the latter.

Reply:
You can use an `env` var to enable/disable dumping at runtime, but all behind a feature guard please :)

Reply:
Unconditional is fine otherwise I think.